tests/test_algorithm.py

```diff
@@ -759,6 +759,48 @@ def test_invalid_order_parameters(self):
         )
         algo.run(self.data_portal)
 
+    def test_cannot_order_in_before_trading_start(self):
+        algotext = """
+from zipline.api import sid
+from zipline.errors import OrderInBeforeTradingStart
```

Comment:
The second part of https://github.com/quantopian/zipline/pull/1165/commits/bc443d78b95be2f8796b201de4ed04268da0d697#r60965562 could also use `nose.parameterized`.